manifests/m/Microsoft/Git/2.36.1.0.0/Microsoft.Git.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Git
+PackageVersion: 2.36.1.0.0
+PackageLocale: en-US
+Publisher: The Git Fundamentals Team at GitHub
+PublisherUrl: https://github.com/microsoft/git
+PublisherSupportUrl: https://github.com/microsoft/git/issues
+Author: Microsoft
+PackageName: Microsoft Git

```

Would this be something you guys could fix so it stands out from Git.Git?
```suggestion
PackageName: Git
```